File: atividade04/python/login1.py
valid_user = "admin"
valid_pass = "1234"
def login(user, password):
    return user == valid_user and password == valid_pass
user = input("Usuario: ")
password = input("Senha: ")
if login(user, password):
    print("Acesso concedido")
else:
    print("Acesso negado")

File: PYTO/go/login1.go
package main 
import (
    "fmt"
 )
var validUser = "admin"
var validPass = "1234"
func login(user, pass string) bool {
    return user == validUser && pass == validPass
}
func main() {
    var user, pass string
    fmt.Print("Usuario: ")
    fmt.Scanln(&user)
    fmt.Print("Senha: ")
    fmt.Scanln(&pass)
    if login(user, pass) {
        fmt.Println("Acesso concedido")
    } else {
        fmt.Println("Acesso negado")
    }
}